tests/lib/rules/indent.js

```diff
@@ -1570,6 +1581,15 @@ ruleTester.run("indent", rule, {
         },
         {
             code:
+            "require('http').request({hostname: 'localhost',\n" +
+            "                  port: 80}, function(res) {\n" +
+            "  res.end();\n" +
+            "});\n",
```

Comment:
It might be good to have an `output` for this test, just to verify that the autofixer handles these cases correctly.